src/test/java/org/rulez/demokracia/pdengine/CalculateWinnersTest.java

```diff
@@ -53,14 +43,40 @@ public void calculate_winners_returns_none_of_the_ignored_choices() {
 	@Test
 	public void calculate_winners_returns_not_ignored_winner() {
 		List<String> winners = computedVote.calculateWinners(Arrays.asList("C"));
-		assertBeatTableContainChoice(winners, "A");
+		assertTrue(winners.contains("D"));
+	}
+
+	@TestedBehaviour("all non-beaten candidates are winners")
+	@Test
+	public void calculate_winners_doesnt_return_beaten_candidates() {
+		assertTrue(noWinnerIsBeaten(winnerCalculator.calculateWinners(beatPathTable)));
```

Comment:
You could simply create assertNoWinnerIsBeaten and later assertNonbeatenAreWinners...
Acceptable as is, but might be better.